packages/rancher-windows-exporter/charts/templates/configmap.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "windowsExporter.name" . }}-scripts
+  namespace: {{ template "windowsExporter.namespace" . }}
+  labels: {{ include "windowsExporter.labels" . | nindent 4 }}
+data:
+  copy-binary.ps1: |-
+{{ .Files.Get "scripts/copy-binary.ps1" | indent 4 }}
+  proxy-entry.ps1: |-
+{{ .Files.Get "scripts/proxy-entry.ps1" | indent 4 }}
```

Comment:
This part of the file is new, the copy-binary script is a copy from v1 https://github.com/rancher/system-charts/blob/3e8ab13429caede329976f7f556b3cac8b0421cd/charts/rancher-monitoring/v0.2.1/charts/exporter-node-windows/templates/configmap.yaml#L11